packages/terra-form-select/src/_Frame.module.scss

```diff
@@ -267,6 +266,30 @@
     }
   }
 
+  .tag {
+    > .display {
+      padding-bottom: var(--terra-search-select-padding-bottom, 0.21428571428571427rem);
+    }
+  }
+
+  .search {
+    > .display {
+      padding-bottom: var(--terra-search-select-padding-bottom, 0.21428571428571427rem);
+    }
+  }
+
+  .combobox {
+    > .display {
+      padding-bottom: var(--terra-combobox-select-padding-bottom, 0.21428571428571427rem);
+    }
+  }
+  
+  .multiple {
+    > .display {
+      padding-bottom: var(--terra-multiple-select-padding-bottom, 0.21428571428571427rem);
+    }
+  }
```

Comment:
Thanks for working on this Mahesh. I have a few concerns about this solution. My initial concern is creating duplicate / identical variables for each variant. I believe the Select component should be themed by content rather than by the variant type. This solution opens the door to individual variant theming. Meaning the combobox, multiple, search, and tag could all display with different amounts of padding, color, and other styles. And requires theming 4 variables to meet a singular common goal. If we were to expand this concept to the other variables this Select component would grow to have over 600 variables to theme each variant uniquely. 

Part of the problem could potentially be solved by adding a theme-able padding bottom to the tags wrapper. https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/_Frame.module.scss#L123

This would theme the wrapping content of tags for multiple and tag variants. The other part of this problem is themeing the padding / margin for search and combobox variants. This may require adding a content div around the input or theming the input slightly differently to account for the change in margins / padding.

Reply:
Here is an example of another potential solution, this adds a wrapping div around the `combobox` and `search` variants and allows them to be themed together with `tag` and `multiple` variants. A better name than `content` should be chosen.

https://github.com/cerner/terra-core/commit/b75b5bcf1807128dd05e298f32aafc2d96d3eefe